manala.nginx/templates/configs/app_magento_2_default.j2

```diff
@@ -0,0 +1,174 @@
+## Optional override of deployment mode. We recommend you use the
```

Comment:
Considering that drupal default configuration template (https://github.com/manala/ansible-roles/blob/master/manala.nginx/templates/configs/app_drupal_8_defaults.j2) is called `defaults` (with an `s`), either magento or drupal template should be renamed